Anlab.Mvc/Services/DbInitializationService.cs

```diff
@@ -49,6 +49,33 @@ public DbInitializationService(ApplicationDbContext context, UserManager<User> u
                 "AL-KCL",
                 "AL-OX",
                 "AL-PY",
+                "-BCL-P-IC",
```

Comment:
@jSylvestre i know this is just test data so w/e, but I think the ones that start with dashes are not valid codes, they are like for QC or tests or something